python/pyarrow/orc.py

```diff
@@ -330,7 +330,9 @@ def read_table(source, columns=None, filesystem=None):
 """
 
 
-def write_table(table, where, *, file_version='0.12',
+def write_table(table, where, *,
+                close_file=False,
```

Comment:
We don't add this argument after all, do we?

Reply:
This argument does serve a real ORC purpose (please see the tests in `test_orc.py).` It’s not the `filesystem` argument I removed.

Reply:
The argument is no longer passed down, though. So I don't think it can do something

Reply:
@jorisvandenbossche It is still being passed down in `orc.py` and `_orc.pyx`. May I ask where you see a static `file_version` being used in the code? Really thanks!

Reply:
The comment is about `close_file`, not `file_version` keyword